docs/server-side/v2/introduction/webhook.mdx

```diff
@@ -1078,25 +1086,27 @@ This event will be sent when HLS recordings are successful and uploaded to the s
+| duration                            | `number`             | Duration of HLS recording in seconds <br/><br/> _Example: 12 _                                                                     |
+| session_id                          | `string`             | 100ms assigned id to identify the session <br/><br/> _Example: 5f9edc6bd238215aec7700df _                                          |
+| template_id                         | `string`             | Template ID of the room <br/><br/> _Example: 66112497abcd52312556c4gg _                                                            |
+| hls_vod_recording_path              | `string`             | Upload path of the HLS vod playlist such as s3 URI <br/><br/> _Example: s3://bucket/prefix/ac.mp4 _                                |
+| hls_vod_recording_presigned_url     | `string`             | Pre signed url for HLS vod playlist url <br/><br/> _Example: https://s3-location/bucket/hls-vod.zip _                              |
+| max_width                           | `int`                | Maximum width of the screen supported for HLS recording in pixels <br/><br/> _Example: 1280 _                                      |
+| max_height                          | `int`                | Maximum height of the screen supported for HLS recording in pixels <br/><br/> _Example: 720 _                                      |
+| meeting_url                         | `string`             | meeting_url provided at HLS start <br/><br/> Example: "https://app.100ms.live/room_id"                                             |
+| recording_created_at                | `timestamp (in UTC)` | Timestamp at which recording was created <br/><br/> _Example: 2020-11-11T17:12:17Z _                                               |
+| started_at                          | `timestamp (in UTC)` | Timestamp at which HLS started <br/><br/> _Example: 2020-11-11T17:32:18Z _                                                         |
+| stopped_at                          | `timestamp (in UTC)` | Timestamp at which HLS stopped <br/><br/> _Example: 2020-11-11T17:32:18Z _                                                         |
+| recording_single_files              | `array`              | List of recording details per layer. layer="0" is the highest quality layer <br/><br/> _Example: < see below > _                   |
+| recording_hls_vod_playlist_location | `string`             | Pre signed url for HLS vod playlist url <br/><br/> _Example: < see below > _                                                       |
+| recording_thumbnails                | `array`              | List of thumbnails generated <br/><br/> _Example: < see below > _                                                                  |
+| session_started_at                  | `timestamp (in UTC)` | Timestamp when session started <br/><br/> _Example: 2020-11-11T16:32:17Z _                                                         |
 
 #### Sample `hls.recording.success` event
```

Comment:
track.recording.<> related are not added yet?

Reply:
yes, I couldn't find them here. cc: @sagargaonkar @AnshulMalik

Reply:
@Nixon-Samuel As discussed, let's track this separately